components/header/README.md

```diff
@@ -25,20 +38,7 @@ Good examples include:
 *	Renew your passport
 *	Find an apprenticeship
 
-*Adapted from the UK Government’s Service Naming Guide [https://www.gov.uk/service-manual/design/naming-your-service]
-
-### Required
-
-## Last Updated: 2 days ago
-
-## Visual
-Header Visual:
-![Header](https://github.com/bcgov/design-system/blob/master/components/header/images/header.png?raw=true)
-
-## Use This For
-
-*	All public-facing online B.C. Government content and services.
-*	Websites or apps that do not require navigation built into the header.
+*Adapted from the UK Government’s [Service Naming Guide] (https://www.gov.uk/service-manual/design/naming-your-service)
```

Comment:
remove the white space between `[Service Naming Guide] (https://www.gov.uk/service-manual/design/naming-your-service)`